users/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class UsersConfig(AppConfig):
     name = 'users'
+
+    def ready(self):
+        import users.signals # noqa
```

Comment:
Adding # noqa to the line indicates that flake8 should not check this line.
That line may have something that "looks bad" to the flake8, and gave the warning 'imported but unused'.
The django documentation recommends do it that way to avoid some side effect of how import works.

> https://flake8.pycqa.org/en/latest/user/violations.html#in-line-ignoring-errors

Reply:
+1 thanks